tools/verifygitlog.py

```diff
@@ -107,7 +107,7 @@ def verify_message_body(raw_body, err):
             err.error("Message lines should be 75 or less characters: " + line)
 
     if not raw_body[-1].startswith("Signed-off-by: ") or "@" not in raw_body[-1]:
-        err.warning('Message should be signed-off. Use "git commit -s".')
+        err.error('Message should be signed-off. Use "git commit -s".')
```

Comment:
```suggestion
        err.error('Message must be signed-off. Use "git commit -s".')
```

Reply:
Yes I almost make this change myself but then noticed the wording for all other errors uses "should" (eg "Second message line should be empty").

So either stick with "should" everywhere, or change everything to use "must".

Reply:
If there are exceptions to the rule, the "should" seems appropriate. If there are no exceptions to the rule then "must" seems appropriate. Since you said that SOB is now a strict requirement, "must" seems like the appropriate language (as in RFC semantics).

Reply:
OK, I've replaced "should" with "must" in all places where it's appropriate.